src/swap/exolix.js

```diff
@@ -136,34 +147,33 @@ export function makeExolixPlugin(opts: EdgeCorePluginOptions): EdgeSwapPlugin {
         toMainnetCode
       } = getCodes(request)
 
-      // The Exolix documentation doesn't detail this mainnetCode:currencyCode functionality
-      // but it's been verified by testing
-      const safeFromCurrencyCode = `${fromMainnetCode}:${fromCurrencyCode}`
-      const safeToCurrencyCode = `${toMainnetCode}:${toCurrencyCode}`
-
       // Swap the currencies if we need a reverse quote:
       const quoteParams =
         request.quoteFor === 'from'
           ? {
-              coin_from: safeFromCurrencyCode,
-              coin_to: safeToCurrencyCode,
-              deposit_amount: quoteAmount,
-              rate_type: 'fixed'
+              coinFrom: fromMainnetCode,
```

Comment:
You don't differentiate between the from and to amounts. If `request.quoteFor === 'to'` then the user is specifying the amount of crypto to receive and will be sending the `fromMainnetCode` currency. The code below just assumes from and to are in reverse. Major bug! Bug exists in current plugin which I'll have to fix for now to not allow reverse quotes.